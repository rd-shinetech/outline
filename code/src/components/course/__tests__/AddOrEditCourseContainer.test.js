import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import {AddOrEditCourseContainer} from '../AddOrEditCourseContainer';


describe('AddOrEditCourseContainer.test.js', () => {

    it('renders without crashing', () => {
        const props = {
            action: {
                getCourseAction: jest.fn(),
                getAuthorsAction: jest.fn()
            },
            authors:[],
            initialValues: {id: '', watchHref: '', title: '', authorId: '', length: '', category: ''},
            match: {params: {id:'1'}}
        };

        const wrapper = shallow(<AddOrEditCourseContainer {...props}/>);
        expect(wrapper.length).toEqual(1);

        const tree = toJson(wrapper);
        expect(tree).toMatchSnapshot();
    });

});



    
        