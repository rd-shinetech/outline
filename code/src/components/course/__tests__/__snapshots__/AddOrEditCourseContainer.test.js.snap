exports[`AddOrEditCourseContainer.test.js renders without crashing 1`] = `
<div
  className="container">
  <ReduxForm
    authors={Array []}
    handleCancel={[Function]}
    handleSave={[Function]}
    heading="Add"
    initialValues={
      Object {
        "authorId": "",
        "category": "",
        "id": "",
        "length": "",
        "title": "",
        "watchHref": "",
      }
    } />
</div>
`;
